### PR TITLE
Fix model cut of by bottom positioning

### LINKED ISSUE #1416: Unable to scroll to the top of the modal when using a long text + position: 'bottom-end'
Hi guys,

I still have the issue if I'm using any bottom positioning. 

Tested with the last release (8.2.0) on different browsers (like FF, Chrome, Safari): 
https://jsfiddle.net/1sLmb7j2/

_Originally posted by @SchoppAx in https://github.com/sweetalert2/sweetalert2/issues/679#issuecomment-464709016_

### PR DESCRIPTION
**Pull request by SchoppAx**
Fixes #1416 

Test: https://jsfiddle.net/bqv0pzku/

### CODE REVIEW

File: src/sweetalert2.scss [hunk 1]
```diff
@@ -160,6 +160,14 @@ body {
     align-items: flex-end;
     justify-content: flex-end;
   }
+
```
Comment (limonte) [inline src/sweetalert2.scss]: trailing spaces here

Comment (limonte) [reply src/sweetalert2.scss]: tip: if you're using VSCode, the setting is `"files.trimTrailingWhitespace": true`

Comment (SchoppAx) [reply src/sweetalert2.scss]: I will change the request in a few hours. I‘m currently not at home.

**Comment by limonte** [comment]
Congrats @SchoppAx you're now the contributor of this repo 🎉

**Pull request merged**